### PR TITLE
#31 Restaurant Class

### PR DESCRIPTION
**Pull request by YannisAm**
I created the Restaurant class and i added a migration. I think its fine

### CODE REVIEW

**Comment by YannisAm** [comment]
I did not add a DTO. Do i have to do it manually?

**Comment by chrisK00** [comment]
> I did not add a DTO. Do i have to do it manually?

you can make a new issue for dto like i did with the menu

**Comment by YannisAm** [comment]
So DTOs are made afterwards

File: CRUDRecipeEF.BL.DL/Entities/Restaurant.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Threading.Tasks;
+
+namespace CRUDRecipeEF.BL.DL.Entities
+{
+    public class Restaurant
+    {
+        public int Id { get; set; }
+        [Required]
+        public string RestauarantName { get; set; }
+        public List<Recipe> Recipes { get; set; } = new List<Recipe>();
```
Comment (chrisK00) [inline CRUDRecipeEF.BL.DL/Entities/Restaurant.cs:15]: The restaurant has a menu, dont need recipes and ingredients
Also considering the class name is restaurant and to be consistent with our naming the RestaurantName property can be named only Name

**Review by chrisK00 — CHANGES_REQUESTED** [review]

File: CRUDRecipeEF.BL.DL/Entities/Restaurant.cs (hunk 1 above)
Comment (YannisAm) [reply CRUDRecipeEF.BL.DL/Entities/Restaurant.cs:15]: I try to add:

public IMainMenu MainMenu {get; set;}
but it complaints on IMainMenu

Comment (chrisK00) [reply CRUDRecipeEF.BL.DL/Entities/Restaurant.cs]: oh wait lol. I meant the recipeMenu but we dont have it implemented yet issue #30 
You either add a todo comment and then make an issue for it or just implement the menu then restaurant

Comment (YannisAm) [reply CRUDRecipeEF.BL.DL/Entities/Restaurant.cs]: Hahahaha. Yeah. I got it afterwards, that something was missing and it was easy. silly mistake from me

**Pull request closed without merging**